src/commands/etherfi.ts

```diff
@@ -294,6 +316,39 @@ async function getKeyshareObjects(
   return keysharesObjectsList;
 }
 
+async function areKeysharesValid(keysharesObjArray:ShareObject[], ownerNonce: number, owner: string) {
+
+  let k = new KeySharesItem();
```

Comment:
Should we rename this variable 'k' to something else? Maybe **keySharesValidator**, but might be too long